searchcore/src/vespa/searchcore/proton/server/documentbucketmover.h

```diff
@@ -6,48 +6,113 @@
 #include <vespa/document/base/globalid.h>
 #include <persistence/spi/types.h>
 
+namespace vespalib { class IDestructorCallback; }
+
 namespace proton {
 
 class BucketDBOwner;
 struct IDocumentMoveHandler;
 struct IMoveOperationLimiter;
 class MaintenanceDocumentSubDB;
+class MoveOperation;
 
-/**
- * Class used to move all documents in a bucket from a source sub database
- * to a target sub database. The actual moving is handled by a given instance
- * of IDocumentMoveHandler.
- */
-class DocumentBucketMover
+namespace bucketdb {
+    /**
+  * Class used to move all documents in a bucket from a source sub database
```

Comment:
This description must be updated. It is currently identical to the description in DocumentBucketMover.